tools/releaser/breaking_changes/v20240530003.md

```diff
@@ -0,0 +1,32 @@
+# Breaking Changes
+
+## SDK changes
+
+go.mongodb.org/atlas-sdk/v20240530003/admin
+Incompatible changes:
+
+- (\*BillingInvoice).GetGroupId: removed
+- (\*BillingInvoice).GetGroupIdOk: removed
+- (\*BillingInvoice).HasGroupId: removed
+- (\*BillingInvoice).SetGroupId: removed
+- (\*BillingInvoiceMetadata).GetGroupId: removed
+- (\*BillingInvoiceMetadata).GetGroupIdOk: removed
+- (\*BillingInvoiceMetadata).HasGroupId: removed
+- (\*BillingInvoiceMetadata).SetGroupId: removed
+- BillingInvoice.GroupId: removed
+- BillingInvoiceMetadata.GroupId: removed
```

Comment:
Investigating cause. We do not have this covered in changelog and breaking changes report.

Reply:
This was optional value in response that were not needed.